plexpy/notifiers.py

```diff
@@ -2390,7 +2399,7 @@ def agent_notify(self, subject='', body='', action='', **kwargs):
         logger.info("Tautulli Notifiers :: Sending {name} notification...".format(name=self.NAME))
 
         paho.mqtt.publish.single(
-            self.config['topic'], payload=json.dumps(data), qos=self.config['qos'], retain=bool(self.config['retain']),
+            topic, payload=data, qos=self.config['qos'], retain=bool(self.config['retain']),
```

Comment:
Still need `json.dumps`

```suggestion
            topic, payload=json.dumps(data), qos=self.config['qos'], retain=bool(self.config['retain']),
```